app/scripts/components/modelcontrolmenu.js

```diff
@@ -11,18 +11,58 @@ var exports = (function () {
 
       return {
         collapseShow: true,
+        selectedDownloads: {
```

Comment:
A bit specific. Can we generalize this so we can easier include other models later?